tests/queries/0_stateless/03560_validate_max_insert_block_size.sql

```diff
@@ -0,0 +1,8 @@
+DROP TABLE IF EXISTS t;
```

Comment:
As a general rule, we want people to understand what the test is about at a high level _without_ having them read the whole thing (In this case, it is just a few lines so it's okay).

Therefore, please add a brief comment about the test purpose on top, alternatively a reference to the GitHub issues.

Reply:
Sure, I'll add.

Reply:
Added a reference to the github issue.